packages/data/tests/unit/adapters/dimensions/keg-test.js

```diff
@@ -80,27 +80,42 @@ module('Unit | Adapters | Dimensions | Keg', function(hooks) {
     });
   });
 
-  test('find', function(assert) {
-    assert.expect(2);
-
-    assert.throws(
-      () => {
-        Adapter.find('dimensionOne', { operator: 'contains' });
-      },
-      /Only 'in' operation is currently supported in Keg/,
-      'throws error when doing a contains search, which is not supported yet'
-    );
+  test('find', async function(assert) {
+    assert.expect(8);
+
+    const assertThrowOperator = query => {
+      assert.throws(
+        () => {
+          Adapter.find('dimensionOne', query);
+        },
+        /Only 'in' operation is currently supported in Keg/,
+        'throws error when doing a contains search, which is not supported yet'
+      );
+    };
+    assertThrowOperator({ operator: 'contains' });
+    assertThrowOperator([{ operator: 'contains' }]);
+    assertThrowOperator([{}, { operator: 'in' }, { operator: 'contains' }]);
 
-    return Adapter.find('dimensionOne', {
-      field: 'description',
-      values: 'bar,gar'
-    }).then(result => {
+    const assertPromise = expected => result => {
```

Comment:
It would be nice to add another argument after expected for the assert message. That way we can have a unique message for each assertion.